std/io.zig

```diff
@@ -232,9 +232,9 @@ pub fn OutStream(comptime WriteError: type) type {
             return self.writeFn(self, slice);
         }
 
-        pub fn writeByteNTimes(self: *Self, byte: u8, n: usize) Error!void {
+        pub fn writeByteNTimes(self: *Self, byte: u8, n: u64) Error!void {
```

Comment:
Hmmm. I'd like to push back on this one. Doing an action N times is more like looping over an array - you can see the loop below in fact - and array lengths are `usize`. I think this one should stay `usize`.

Reply:
Roger that.